feconf.py

```diff
@@ -83,6 +83,10 @@
     'TextInput': {
```

Comment:
I'm fine waiting till later to drop this, but it seems a bit odd that tests rely on it. We should probably update the tests to swap this out with a fake value and have a backend test that verifies that every element of this dict has the same structure as the fake value (to prevent skew). I'm agnostic with regards to whether to do this now or later.